### ISSUE TITLE
Package Compatibility Issue with Flutter 3.22.0

### ISSUE THREAD

**Issue by guplem** [GetStream/stream-chat-flutter#1920]
### Which packages are you using?

stream_chat_flutter

### On what platforms did you experience the issue?

iOS, Android

### What version are you using?

`7.2.0` and `8.0.0-beta.3`

### What happened?

The `stream_chat_flutter` package, is causing a build failure when compiling with Flutter 3.22.0. 
The issue is due to its dependency on `dart_vlc: 0.4.0`, which uses `audio_video_progress_bar v0.11.0`. 
The `audio_video_progress_bar` package in its version `0.11.0` is incompatible with Flutter 3.22.0 because it references the deprecated `bodyText1` getter in `TextTheme`.

### Steps to reproduce

```bash
1. Update Flutter to version 3.22.0.
2. Attempt to build a project that includes the `stream_chat_flutter` package.
3. Observe the build failure with the provided error logs.
```

### Supporting info to reproduce

**Additional Information:**
The issue is due to the `dart_vlc` package, which is discontinued and replaced by `media_kit`. The `dart_vlc` package relies on `audio_video_progress_bar v0.11.0`, which references the deprecated `bodyText1` getter in `TextTheme`.

**Proposed Solution:**
- Update `stream_chat_flutter` to replace the `dart_vlc` dependency with `media_kit`.
- Alternatively, update the `dart_vlc` package (if feasible) to use a compatible version of `audio_video_progress_bar` or modify the dependency to avoid using the deprecated `bodyText1` getter.

### Relevant log output

```shell
Build Logs Output:

"Install Android SDK Platform 34 (revision: 3)" finished.
../.pub-cache/hosted/pub.dev/audio_video_progress_bar-0.11.0/lib/audio_video_progress_bar.dart:259:61: Error: The getter 'bodyText1' isn't defined for the class 'TextTheme'.
 - 'TextTheme' is from 'package:flutter/src/material/text_theme.dart' ('../programs/flutter/packages/flutter/lib/src/material/text_theme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'bodyText1'.
    final textStyle = timeLabelTextStyle ?? theme.textTheme.bodyText1;
                                                            ^^^^^^^^^
../.pub-cache/hosted/pub.dev/audio_video_progress_bar-0.11.0/lib/audio_video_progress_bar.dart:290:61: Error: The getter 'bodyText1' isn't defined for the class 'TextTheme'.
 - 'TextTheme' is from 'package:flutter/src/material/text_theme.dart' ('../programs/flutter/packages/flutter/lib/src/material/text_theme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'bodyText1'.
    final textStyle = timeLabelTextStyle ?? theme.textTheme.bodyText1;
                                                            ^^^^^^^^^
Target kernel_snapshot failed: Exception


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileFlutterBuildRelease'.
> Process 'command '/Users/builder/programs/flutter/bin/flutter'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3m 29s
Running Gradle task 'bundleRelease'...                            210.7s
Gradle task bundleRelease failed with exit code 1

Build failed :|
Failed to build for Android
```
```


### Flutter analyze output

```shell
info - 'appBarHeight' is deprecated and shouldn't be used. Use SafeArea instead - lib\V2\components\custom_app_bar_v2.dart:67:45 -
          deprecated_member_use_from_same_package
   info - 'appBarHeight' is deprecated and shouldn't be used. Use SafeArea instead - lib\V2\components\custom_app_bar_v2.dart:69:38 -
          deprecated_member_use_from_same_package
   info - 'Styles' is deprecated and shouldn't be used. Use the new `ThemeV2` - lib\V2\components\video_background_widget.dart:45:24 -
          deprecated_member_use_from_same_package
   info - 'Styles' is deprecated and shouldn't be used. Use the new `ThemeV2` - lib\V2\components\video_background_widget.dart:112:28 -
          deprecated_member_use_from_same_package
   info - The imported package 'http' isn't a dependency of the importing package - lib\V2\data_engine\clients\rest_client.dart:15:8 - depend_on_referenced_packages
   info - The declaration '_getObjectsLoading' isn't referenced - lib\V2\data_engine\components\loader.dart:126:16 - unused_element
   info - The file name 'iService.dart' isn't a lower_case_with_underscores identifier - lib\V2\iService.dart:1:1 - file_names
   info - The type name 'iService' isn't an UpperCamelCase identifier - lib\V2\iService.dart:4:16 - camel_case_types
   info - 'deviceId' is deprecated and shouldn't be used. Get the device ID from the DeviceDataRepository - lib\V2\modules\analytics\analytics_service.dart:117:41 -    
          deprecated_member_use_from_same_package
   info - 'deviceId' is deprecated and shouldn't be used. Get the device ID from the DeviceDataRepository - lib\V2\modules\analytics\analytics_service.dart:143:43 -    
          deprecated_member_use_from_same_package
   info - 'deviceId' is deprecated and shouldn't be used. Get the device ID from the DeviceDataRepository - lib\V2\modules\analytics\analytics_service.dart:164:42 -    
          deprecated_member_use_from_same_package
   info - 'MaterialStateProperty' is deprecated and shouldn't be used. Use WidgetStateProperty instead. Moved to the Widgets layer to make code available outside of    
          Material. This feature was deprecated after v3.19.0-0.3.pre - lib\V2\modules\auth\pages\auth_page.dart:628:53 - deprecated_member_use
   info - 'MaterialStateProperty' is deprecated and shouldn't be used. Use WidgetStateProperty instead. Moved to the Widgets layer to make code available outside of    
          Material. This feature was deprecated after v3.19.0-0.3.pre - lib\V2\modules\auth\pages\auth_page.dart:800:85 - deprecated_member_use
   info - 'appBarHeight' is deprecated and shouldn't be used. Use SafeArea instead - lib\V2\modules\chat_v3\components\appbar\chat_appbar.dart:44:45 -
          deprecated_member_use_from_same_package
   info - 'MaterialStateProperty' is deprecated and shouldn't be used. Use WidgetStateProperty instead. Moved to the Widgets layer to make code available outside of    
          Material. This feature was deprecated after v3.19.0-0.3.pre - lib\V2\modules\chat_v3\components\input_bar\media_menu.dart:29:20 - deprecated_member_use       
   info - 'MaterialStateProperty' is deprecated and shouldn't be used. Use WidgetStateProperty instead. Moved to the Widgets layer to make code available outside of    
          Material. This feature was deprecated after v3.19.0-0.3.pre - lib\V2\modules\chat_v3\components\input_bar\media_menu.dart:30:26 - deprecated_member_use       
   info - 'fetchOne' is deprecated and shouldn't be used. Use [fetch] instead - lib\V2\modules\chat_v3\repository.dart:38:43 - deprecated_member_use_from_same_package  
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\chat_v3\service.dart:129:7 - use_build_context_synchronously
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\chat_v3\service.dart:130:21 - use_build_context_synchronously
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\chat_v3\service.dart:131:23 - use_build_context_synchronously
   info - 'background' is deprecated and shouldn't be used. Use surface instead. This feature was deprecated after v3.18.0-0.1.pre -
          lib\V2\modules\chat_v3\theme.dart:19:34 - deprecated_member_use
   info - 'background' is deprecated and shouldn't be used. Use surface instead. This feature was deprecated after v3.18.0-0.1.pre -
          lib\V2\modules\chat_v3\theme.dart:20:35 - deprecated_member_use
   info - 'background' is deprecated and shouldn't be used. Use surface instead. This feature was deprecated after v3.18.0-0.1.pre -
          lib\V2\modules\chat_v3\theme.dart:41:36 - deprecated_member_use
   info - 'onBackground' is deprecated and shouldn't be used. Use onSurface instead. This feature was deprecated after v3.18.0-0.1.pre -
          lib\V2\modules\chat_v3\theme.dart:48:45 - deprecated_member_use
   info - The imported package 'http' isn't a dependency of the importing package - lib\V2\modules\create\components\create_modal_gallery.dart:25:8 -
          depend_on_referenced_packages
   info - 'oldServerUrl' is deprecated and shouldn't be used. Use restServerUrl instead - lib\V2\modules\create\components\create_modal_info.dart:436:85 -
          deprecated_member_use_from_same_package
   info - The imported package 'http' isn't a dependency of the importing package - lib\V2\modules\create\components\create_modal_location.dart:23:8 -
          depend_on_referenced_packages
   info - Don't use 'BuildContext's across async gaps, guarded by an unrelated 'mounted' check - lib\V2\modules\create\components\create_modal_location.dart:95:11 -    
          use_build_context_synchronously
   info - 'background' is deprecated and shouldn't be used. Use surface instead. This feature was deprecated after v3.18.0-0.1.pre -
          lib\V2\modules\dev_tools\components\color_palette_demo.dart:86:65 - deprecated_member_use
   info - 'onBackground' is deprecated and shouldn't be used. Use onSurface instead. This feature was deprecated after v3.18.0-0.1.pre -
          lib\V2\modules\dev_tools\components\color_palette_demo.dart:86:98 - deprecated_member_use
   info - 'onBackground' is deprecated and shouldn't be used. Use onSurface instead. This feature was deprecated after v3.18.0-0.1.pre -
          lib\V2\modules\dev_tools\components\color_palette_demo.dart:87:67 - deprecated_member_use
   info - 'background' is deprecated and shouldn't be used. Use surface instead. This feature was deprecated after v3.18.0-0.1.pre -
          lib\V2\modules\dev_tools\components\color_palette_demo.dart:87:102 - deprecated_member_use
   info - 'surfaceVariant' is deprecated and shouldn't be used. Use surfaceContainerHighest instead. This feature was deprecated after v3.18.0-0.1.pre -
          lib\V2\modules\dev_tools\components\color_palette_demo.dart:95:69 - deprecated_member_use
   info - 'surfaceVariant' is deprecated and shouldn't be used. Use surfaceContainerHighest instead. This feature was deprecated after v3.18.0-0.1.pre -
          lib\V2\modules\dev_tools\components\color_palette_demo.dart:96:110 - deprecated_member_use
   info - 'clearAllData' is deprecated and shouldn't be used. Use remove() instead.
This Should not be used because some data should be permanent (like deviceId) and is not supposed to be deleted -
       lib\V2\modules\dev_tools\components\shared_preferences.dart:19:56 - deprecated_member_use_from_same_package
warning - Unused import: 'package:bondy/V2/modules/feed/service.dart' - lib\V2\modules\feed\components\final_page.dart:4:8 - unused_import
   info - 'MaterialStateProperty' is deprecated and shouldn't be used. Use WidgetStateProperty instead. Moved to the Widgets layer to make code available outside of
          Material. This feature was deprecated after v3.19.0-0.3.pre - lib\V2\modules\feed\components\final_page.dart:51:27 - deprecated_member_use
   info - 'MaterialStateProperty' is deprecated and shouldn't be used. Use WidgetStateProperty instead. Moved to the Widgets layer to make code available outside of    
          Material. This feature was deprecated after v3.19.0-0.3.pre - lib\V2\modules\feed\components\fiters_page.dart:55:27 - deprecated_member_use
   info - 'deviceId' is deprecated and shouldn't be used. Get the device ID from the DeviceDataRepository - lib\V2\modules\global\global_data_service.dart:154:45 -     
          deprecated_member_use_from_same_package
   info - 'deviceId' is deprecated and shouldn't be used. Get the device ID from the DeviceDataRepository - lib\V2\modules\global\global_data_service.dart:205:7 -      
          deprecated_member_use_from_same_package
   info - 'deviceId' is deprecated and shouldn't be used. Get the device ID from the DeviceDataRepository - lib\V2\modules\global\global_data_service.dart:208:7 -      
          deprecated_member_use_from_same_package
   info - 'deviceId' is deprecated and shouldn't be used. Get the device ID from the DeviceDataRepository - lib\V2\modules\global\global_data_service.dart:210:7 -      
          deprecated_member_use_from_same_package
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\live\pages\live_page.dart:123:40 - use_build_context_synchronously
   info - 'fetchOne' is deprecated and shouldn't be used. Use [fetch] instead - lib\V2\modules\live\pages\live_page.dart:202:24 -
          deprecated_member_use_from_same_package
   info - 'fetchOne' is deprecated and shouldn't be used. Use [fetch] instead - lib\V2\modules\live\pages\live_page.dart:606:66 -
          deprecated_member_use_from_same_package
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\location\map_page.dart:53:122 - use_build_context_synchronously
   info - The file name 'customButton.dart' isn't a lower_case_with_underscores identifier - lib\V2\modules\people\components\customButton.dart:1:1 - file_names        
   info - The file name 'imagesTile.dart' isn't a lower_case_with_underscores identifier - lib\V2\modules\people\components\imagesTile.dart:1:1 - file_names
   info - The file name 'peopleCard.dart' isn't a lower_case_with_underscores identifier - lib\V2\modules\people\components\peopleCard.dart:1:1 - file_names
   info - The file name 'peoplePlanCard.dart' isn't a lower_case_with_underscores identifier - lib\V2\modules\people\components\peoplePlanCard.dart:1:1 - file_names    
   info - The file name 'searchUsers.dart' isn't a lower_case_with_underscores identifier - lib\V2\modules\people\components\searchUsers.dart:1:1 - file_names
   info - The file name 'topInfoTile.dart' isn't a lower_case_with_underscores identifier - lib\V2\modules\people\components\topInfoTile.dart:1:1 - file_names
   info - 'oldServerUrl' is deprecated and shouldn't be used. Use restServerUrl instead - lib\V2\modules\plan\components\plan_details_location_card.dart:97:72 -        
          deprecated_member_use_from_same_package
   info - 'fetchOne' is deprecated and shouldn't be used. Use [fetch] instead - lib\V2\modules\plan\components\plan_participants_modal.dart:38:24 -
          deprecated_member_use_from_same_package
   info - 'fetchOne' is deprecated and shouldn't be used. Use [fetch] instead - lib\V2\modules\plan\components\plan_participants_modal.dart:89:28 -
          deprecated_member_use_from_same_package
   info - The import of 'package:flutter/cupertino.dart' is unnecessary because all of the used elements are also provided by the import of
          'package:flutter/material.dart' - lib\V2\modules\plan\components\plan_share_modal.dart:21:8 - unnecessary_import
   info - 'fetchOne' is deprecated and shouldn't be used. Use [fetch] instead - lib\V2\modules\plan\components\plan_share_modal.dart:55:41 -
          deprecated_member_use_from_same_package
   info - 'fetchOne' is deprecated and shouldn't be used. Use [fetch] instead - lib\V2\modules\plan\components\plan_share_modal.dart:82:53 -
          deprecated_member_use_from_same_package
   info - 'fetchOne' is deprecated and shouldn't be used. Use [fetch] instead - lib\V2\modules\plan\components\plan_share_modal.dart:86:43 -
          deprecated_member_use_from_same_package
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\plan\components\plan_share_modal.dart:100:25 - use_build_context_synchronously
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\plan\components\plan_share_modal.dart:103:95 - use_build_context_synchronously
   info - 'fetchOne' is deprecated and shouldn't be used. Use [fetch] instead - lib\V2\modules\plan\pages\plan_view.dart:100:24 -
          deprecated_member_use_from_same_package
   info - Don't invoke 'print' in production code - lib\V2\modules\plan\pages\plan_view.dart:222:11 - avoid_print
   info - Empty catch block - lib\V2\modules\push_notifications\service.dart:122:17 - empty_catches
   info - Empty catch block - lib\V2\modules\push_notifications\service.dart:140:17 - empty_catches
   info - The file name 'listTile.dart' isn't a lower_case_with_underscores identifier - lib\V2\modules\settings\components\listTile.dart:1:1 - file_names
   info - The file name 'listTileSwitch.dart' isn't a lower_case_with_underscores identifier - lib\V2\modules\settings\components\listTileSwitch.dart:1:1 - file_names  
   info - The file name 'modal_deleteAccount.dart' isn't a lower_case_with_underscores identifier - lib\V2\modules\settings\components\modal_deleteAccount.dart:1:1 -   
          file_names
   info - 'MaterialStateProperty' is deprecated and shouldn't be used. Use WidgetStateProperty instead. Moved to the Widgets layer to make code available outside of    
          Material. This feature was deprecated after v3.19.0-0.3.pre - lib\V2\modules\settings\components\modal_deleteAccount.dart:121:40 - deprecated_member_use      
   info - 'MaterialState' is deprecated and shouldn't be used. Use WidgetState instead. Moved to the Widgets layer to make code available outside of Material. This     
          feature was deprecated after v3.19.0-0.3.pre - lib\V2\modules\settings\components\modal_deleteAccount.dart:122:30 - deprecated_member_use
   info - The file name 'modal_linkGoogle.dart' isn't a lower_case_with_underscores identifier - lib\V2\modules\settings\components\modal_linkGoogle.dart:1:1 -
          file_names
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\settings\components\modal_linkGoogle.dart:90:87 - use_build_context_synchronously
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\settings\components\modal_linkGoogle.dart:93:47 - use_build_context_synchronously
   info - 'Styles' is deprecated and shouldn't be used. Use the new `ThemeV2` - lib\V2\modules\settings\components\modal_linkGoogle.dart:99:45 -
          deprecated_member_use_from_same_package
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\settings\components\modal_linkGoogle.dart:114:87 - use_build_context_synchronously
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\settings\components\modal_linkGoogle.dart:117:47 - use_build_context_synchronously
   info - 'Styles' is deprecated and shouldn't be used. Use the new `ThemeV2` - lib\V2\modules\settings\components\modal_linkGoogle.dart:123:45 -
          deprecated_member_use_from_same_package
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\settings\page\settings_page.dart:262:121 - use_build_context_synchronously
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\settings\page\settings_page.dart:265:81 - use_build_context_synchronously
   info - 'Styles' is deprecated and shouldn't be used. Use the new `ThemeV2` - lib\V2\modules\settings\page\settings_page.dart:271:79 -
          deprecated_member_use_from_same_package
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\settings\page\settings_page.dart:290:121 - use_build_context_synchronously
   info - Don't use 'BuildContext's across async gaps - lib\V2\modules\settings\page\settings_page.dart:293:81 - use_build_context_synchronously
   info - 'Styles' is deprecated and shouldn't be used. Use the new `ThemeV2` - lib\V2\modules\settings\page\settings_page.dart:299:79 -
          deprecated_member_use_from_same_package
   info - 'background' is deprecated and shouldn't be used. Use surface instead. This feature was deprecated after v3.18.0-0.1.pre -
          lib\V2\modules\theme\theme.dart:47:11 - deprecated_member_use
   info - 'onBackground' is deprecated and shouldn't be used. Use onSurface instead. This feature was deprecated after v3.18.0-0.1.pre -
          lib\V2\modules\theme\theme.dart:48:11 - deprecated_member_use
   info - The import of 'package:bondy/V2/utils/extensions/string.dart' is unnecessary because all of the used elements are also provided by the import of
          'package:bondy/V2/utils/library.dart' - lib\V2\modules\url_data\models\urldata.dart:5:8 - unnecessary_import
   info - The imported package 'http' isn't a dependency of the importing package - lib\V2\modules\url_data\models\urldata.dart:7:8 - depend_on_referenced_packages     
   info - Don't return 'null' from a function with a return type of 'void' - lib\V2\modules\user\components\profile_appbar_options.dart:96:25 -
          avoid_returning_null_for_void
   info - Missing a corresponding override of 'hashCode' - lib\V2\utils\classes\cached_xfile.dart:84:17 - hash_and_equals
   info - The file name 'inputFormat.dart' isn't a lower_case_with_underscores identifier - lib\V2\utils\extensions\inputFormat.dart:1:1 - file_names
   info - 'Styles' is deprecated and shouldn't be used. Use the new `ThemeV2` - lib\global_widgets\custom_app_bar.dart:42:89 - deprecated_member_use_from_same_package  
   info - 'Styles' is deprecated and shouldn't be used. Use the new `ThemeV2` - lib\global_widgets\custom_app_bar.dart:53:73 - deprecated_member_use_from_same_package  
   info - 'Styles' is deprecated and shouldn't be used. Use the new `ThemeV2` - lib\global_widgets\custom_app_bar.dart:73:61 - deprecated_member_use_from_same_package  
   info - 'appBarHeight' is deprecated and shouldn't be used. Use SafeArea instead - lib\global_widgets\custom_app_bar.dart:88:45 -
          deprecated_member_use_from_same_package
   info - 'appBarHeight' is deprecated and shouldn't be used. Use SafeArea instead - lib\global_widgets\custom_tab_bar.dart:25:45 -
          deprecated_member_use_from_same_package
   info - 'appBarHeight' is deprecated and shouldn't be used. Use SafeArea instead - lib\global_widgets\custom_tab_bar.dart:27:38 -
          deprecated_member_use_from_same_package
   info - 'surfaceVariant' is deprecated and shouldn't be used. Use surfaceContainerHighest instead. This feature was deprecated after v3.18.0-0.1.pre -
          lib\global_widgets\custom_tab_bar.dart:78:21 - deprecated_member_use
   info - 'Styles' is deprecated and shouldn't be used. Use the new `ThemeV2` - lib\global_widgets\custom_tab_bar.dart:83:27 - deprecated_member_use_from_same_package  
   info - 'Styles' is deprecated and shouldn't be used. Use the new `ThemeV2` - lib\global_widgets\custom_tab_bar.dart:98:34 - deprecated_member_use_from_same_package  
   info - The imported package 'image' isn't a dependency of the importing package - lib\utils\utils.dart:12:8 - depend_on_referenced_packages
```

### Flutter doctor output

```shell
[√] Flutter (Channel stable, 3.22.0, on Microsoft Windows [Version 10.0.22631.3527], locale en-US)
    • Flutter version 3.22.0 on channel stable at C:\Users\guill\DevFiles\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 5dcb86f68f (6 days ago), 2024-05-09 07:39:20 -0500
    • Engine revision f6344b75dc
    • Dart version 3.4.0
    • DevTools version 2.34.3

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at C:\Users\guill\AppData\Local\Android\sdk
    • Platform android-34, build-tools 34.0.0
    • Java binary at: C:\Program Files\Android\Android Studio\jbr\bin\java
    • Java version OpenJDK Runtime Environment (build 17.0.10+0--11572160)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[!] Visual Studio - develop Windows apps (Visual Studio Build Tools 2019 16.11.31)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools
    • Visual Studio Build Tools 2019 version 16.11.34114.132
    X The current Visual Studio installation is incomplete.
      Please use Visual Studio Installer to complete the installation or reinstall Visual Studio.

[√] Android Studio (version 2023.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.10+0--11572160)

[√] VS Code (version 1.89.0)
    • VS Code at C:\Users\guill\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension can be installed from:
       https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22631.3527]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 124.0.6367.207
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 124.0.2478.97

[√] Network resources
    • All expected network resources are available.

! Doctor found issues in 1 category.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mauriziopinotti** [comment]
Meanwhile, since audio_video_progress_bar didn't really change much between 0.x and 2.x versions, it's possible to just override the dependency like this:

```yaml
dependency_overrides:
  # Waiting for https://github.com/GetStream/stream-chat-flutter/issues/1920
  audio_video_progress_bar: ^2.0.2
```

I checked that the video progress bar still works and it builds with Flutter 3.22.

**Comment by rlee1990** [comment]
Thats not the only issue.
I also get the below and doing an override does not resolve.
```
../../.pub-cache/hosted/pub.dev/stream_chat_flutter_core-7.2.0/lib/src/stream_chat_core.dart:127:45: Error: A value of type 'Stream<List<ConnectivityResult>>' can't be assigned to a variable of type 'Stream<ConnectivityResult>?'.
 - 'Stream' is from 'dart:async'.
 - 'List' is from 'dart:core'.
 - 'ConnectivityResult' is from 'package:connectivity_plus_platform_interface/src/enums.dart' ('../../.pub-cache/hosted/pub.dev/connectivity_plus_platform_interface-2.0.0/lib/src/enums.dart').
      connectivityStream ??= Connectivity().onConnectivityChanged;
```

**Comment by deven98** [comment]
We're currently working on making sure the package is compatible with 3.22, this will likely land on the beta release and if possible stable. In general, we keep our beta for the latest Flutter version and keep the stable compatible with one version prior as breaking changes may block users who are not on the latest version from using the SDK.

**Comment by mauriziopinotti** [comment]
> We're currently working on making sure the package is compatible with 3.22, this will likely land on the beta release and if possible stable. In general, we keep our beta for the latest Flutter version and keep the stable compatible with one version prior as breaking changes may block users who are not on the latest version from using the SDK.

I am a bit worried by "if possible", let's make it a "_when_ possible" shall we? :joy: 
(just kidding of course)

**Comment by hitesh-hashkraft** [comment]
How many days it will take to fix this issue?

**Comment by tarunkhatri** [comment]
I am facing same issue @deven98 when are you planning to release new version with the fix, I am not able to build my project due to this issue.

**Comment by mauriziopinotti** [comment]
> I am facing same issue @deven98 when are you planning to release new version with the fix, I am not able to build my project due to this issue.

@tarunkhatri see my comment https://github.com/GetStream/stream-chat-flutter/issues/1920#issuecomment-2112573149 for a temporary workaround

**Comment by deven98** [comment]
The latest release should fix all compatibility issues with Flutter 3.19/3.22. Thanks for reporting this and feel free to open a new issue if you see any other compatibility issues. Thanks!
